.github/workflows/test_iree.yml

```
@@ -157,7 +163,23 @@ jobs:
             --durations=0 \
             --config-files=${MODELS_CONFIG_FILE_PATH}
 
-      - name: "Running real weights SDXL tests"
+      - name: "Running real weights SDXL prompt encoder tests"
+        id: prompt_encode
```

`encode`, or `encoder`?
```suggestion
        id: prompt_encoder
```

---

encoder probably better